recipe/patches/0010-Improve-logic-for-finding-the-macos-sysroot-for-cond.patch

```diff
@@ -0,0 +1,107 @@
+From 06e16a989f21d8c9ec0382a091f0dd32cb83b1da Mon Sep 17 00:00:00 2001
+From: Chris Burr <christopher.burr@cern.ch>
+Date: Sat, 17 Apr 2021 09:26:17 +0200
+Subject: [PATCH 10/10] Improve logic for finding the macos sysroot for conda
```

Comment:
Can you merge this patch with `recipe/patches/0002-clang-add-conda-specific-env-var-CONDA_BUILD_SYSROOT.patch` and also check SDKROOT before CONDA_BUILD_SYSROOT ?